projects/ion/src/lib/popover/component/popover.component.html

```diff
@@ -1,40 +1,40 @@
 <div
-  [class]="'sup-container-' + ionPopoverPosition"
+  [class]="'ion-popover-sup-container--' + ionPopoverPosition"
```

Comment:
```suggestion
  [class]="'ion-popover__sup-container--' + ionPopoverPosition"
```
You can change "sup-container" to a better name like "wrapper" or you can use just "ion-popover--".

Reply:
changed, thanks for the tip👍